build/containers/Dockerfile.webhook-configuration

```diff
@@ -0,0 +1,54 @@
+ARG WEBHOOK="webhooks/validating/configuration"
```

Comment:
To build this currently, the following must be added to `.dockerignore`:
```
!shared

!webhooks
```

Reply:
Yes, that's correct, thank you.... plus I added this as a member in `Cargo.toml` too.

But, I remembered that this was redundant when I wrote the `http` handler and couldn't recall whether that was because the `http` handler is in a distinct branch or because this was redundant by the automated build.

Reply:
the other rust components build their binaries using the `Run Per-Arch component build for ${{ env.AKRI_COMPONENT }}` action invoked in their component workflows.

then, in the Dockerfile.<component>, they just copy what has been built previously.

that's why they don't add !shared and !<component-src-path> to the dockerignore.

Reply:
in essence, you may be building your component a second time here.  you might just be able to copy the cross-built component in here.

Reply:
Sorry, since this PR is a work in progress, I meant that those components need to be added to `.dockerignore` to get the current PR description to work. You make the better point that instead, the dockerfile for the webhook could be modified to fall in convention with our other rust dockerfiles.

Reply:
`cargo build ...` and then containerize...doh! :disappointed:

Reply:
Yes, thanks both of you! I'm sorry this has been tedious for you both.

I think I've (finally!) understood how your build process works.

Reply:
not remotely tedious ... it is something that isn't documented and isn't intuitive.  just glad that it worked for you!!